### PR TITLE
Feat/initialize db

### PR DESCRIPTION
**Pull request by dshelters**
Related to Trello Ticket "implement postgres"
Does not finalize the creation of all schema's and queries, but build the users and rooms tables on server spin up, along with granting access to many basic queries for the two tables.

### CODE REVIEW

File: database/repos/chained.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const tables = {
+  users: require('../queries').users,
+  rooms: require('../queries').rooms
+};
+
+
+module.exports = (repo, pgp) => ({
+  initializeDB: () => (
+      repo.tx((t) => {
```
Comment (jefQuery) [inline database/repos/chained.js]: I would like some explanation on what both repo and tx is, what it's doing, and where it's coming from.

File: database/database.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  rooms: require('./repos/rooms.js'),
+  users: require('./repos/users.js'),
+  chained: require('./repos/chained.js')
+};
+
+const options = {
+  extend: (obj) => {
+    const keys = Object.keys(repos);
+    keys.forEach((key) => {
+      obj[key] = repos[key](obj, pgp);
```
Comment (jefQuery) [inline database/database.js]: Would also like some comments here in the functional programming with slightly vague names, which, while quite clever, is hard to grok.

**Comment by dshelters** [comment]
[implement postgres](https://trello.com/c/Bo185e3q/67-implement-postgres)

File: database/repos/chained.js (hunk 1 above)
Comment (jefQuery) [inline database/repos/chained.js]: Thanks! Merging

**Pull request merged**